### ISSUE TITLE
Feature request: Keyboard support. Press / to focus on search box

### ISSUE THREAD

**Issue by cool-RR** [Nagasaki45/krihelinator#180]

**Comment by Nagasaki45** [comment]
Hi @cool-RR! Thanks for the suggestion. Is that any different from #179 (except for "/" instead of tab)?
I didn't work on this project for a long time, so can't promise anything, but if you want to contribute you're more than welcome!

**Comment by cool-RR** [comment]
Yes, it is separate. "/" is used when focus is in the webpage. Try it on
GMail, GitHub, or Facebook. Pressing tab is done in the address bar, before
you even pressed enter on the url.
